src/service/video-manager-impl.js

```diff
@@ -755,11 +876,64 @@ class VideoEntry {
    *
    * @private
    */
-  unDockVideo_() {
+  finishDocking_() {
     // Restore the video inline
     this.internalElement_.classList.remove(DOCK_CLASS);
     this.internalElement_.setAttribute('style', '');
+    this.dockingState_ = DockingStates.INLINE;
     this.video.showControls();
```

Comment:
Disabled hiding controls when docking begins, the behavior will be fixed when the draggable mask is merged (videos won't be able to capture `mouseover` / `click` events until they're undocked so no way for controls to be shown)